skfda/preprocessing/registration/validation.py

```diff
@@ -111,15 +139,14 @@ class AmplitudePhaseDecomposition(RegistrationScorer):
 
     .. math::
         \text{MSE}_{phase}=
-        \int \left [C_R \overline{y}^2(t) - \overline{x}^2(t) \right]dt
+        C_R \int \overline{y}^2(t) dt - \int \overline{x}^2(t) dt
 
     where the constant :math:`C_R` is defined as
 
     .. math::
 
-        C_R = 1 + \frac{\frac{1}{N}\sum_{i}^{N}\int [Dh_i(t)-\overline{Dh}(t)]
-        [ y_i^2(t)- \overline{y^2}(t) ]dt}
-        {\frac{1}{N} \sum_{i}^{N} \int y_i^2(t)dt}
+        C_R = \frac{\frac{1}{N}\sum_{i=1}^{N}\int[x_i(t)-\overline x(t)]^2dt
```

Comment:
Are the Cr formulas equivalent or was the previous one wrong?
If they are equivalent, it is a great simplification avoiding the use of h(t).

Reply:
The previous formula is derived from this one in the original paper. I think that they derive it for theoretical reasons (in order to know how the warpings influence the decomposition) but for the implementation I think the original one is better as you do not need the warpings.

Reply:
I didn't see that part when I implemented it. 👍